src/tests/runTest.ts

```diff
@@ -18,13 +18,34 @@
 // All tests ran here are ones that require the vscode API
 import * as path from 'path'
 import * as cp from 'child_process'
+import { parseArgs } from 'util'
 import {
   runTests,
   resolveCliArgsFromVSCodeExecutablePath,
   downloadAndUnzipVSCode,
 } from '@vscode/test-electron'
 
 async function main() {
+  const {
+    values: { disable_cert_verification },
+  } = parseArgs({
+    options: {
+      disable_cert_verification: {
+        type: 'boolean',
+        short: 'k',
+        default: false,
+      },
+    },
+  })
+  if (disable_cert_verification) {
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+  }
+
+  // The version of VS Code to use for running the test suite in
+  const testVsCodeVersion = process.env.TEST_VSCODE_VERSION
```

Comment:
Should this be `DAFFODIL_TEST_VSCODE_VERSION` or something that makes it more clear this is Daffodil specific?

Reply:
I think Daffodil is implied in the context, but I'm willing to rename this per your recommendation since it can be exported as a global environment variable (which I would _discourage_ using that way, but instead do it like `TEST_VSCODE_VERSION=1.68.0 node ./out/tests/runTest.js`).

Reply:
Yeah, using is a global env is my concern. I could imagine a developer setting it globally to "stable" or "nightly" or something, and months later looking at their environment variables and not remembering that TEST_VSCODE_VERSION is actually configuring the Daffodil extension tests, but instead thinking it's something that VS Code itself uses or something.

Reply:
I made this change.